### ISSUE TITLE
Yarn: error due to `bn` not being declared as a dependency

### ISSUE THREAD

**Issue by drewgingerich** [renovatebot/kbpgp#84]
I have a project using Yarn PnP and I get the following error when trying to run the `renovate` CLI via the npm package:

```
Error: @renovatebot/kbpgp tried to access bn, but it isn't declared in its dependencies; this makes the require call ambiguous and unsound.
```

This error appears to indicate that `bn` should be declared as a `dependency`, rather than how it's currently declared as a `devDependency`. I'm not sure what the intended setup is, so maybe a `peerDependency` could be more appropriate, too.

Yarn PnP is extremely strict, so it tends to uncover configuration errors that silently work in other setups. 

My `package.json` is as follows, with a few fields redacted:

```json
{
  "scripts": {
    "renovate": "RENOVATE_LOG_FILE=renovate.log RENOVATE_CONFIG_FILE=renovate-config.js renovate",
  },
  "dependencies": {
    "renovate": "^39.20.1",
    "uuid": "^11.0.2",
    "zod": "^3.23.8"
  },
  "devDependencies": {
    "@types/node": "^22.0.0",
    "typescript": "^5.6.3"
  },
  "packageManager": "yarn@4.5.1"
}
```

My `yarnrc.yml` is as follows:

```yml
﻿compressionLevel: mixed

enableGlobalCache: false

supportedArchitectures:
  os:
    - darwin
    - linux
  cpu:
    - x64
    - arm64

yarnPath: .yarn/releases/yarn-4.5.1.cjs
```

**Comment by viceice** [comment]
fixed in 4299fcb7f7760d47469945bf6543e5c4e2b60cc0

**Comment by viceice** [comment]
- https://github.com/renovatebot/renovate/pull/32619
